.github/workflows/ci.yml

```diff
@@ -9,12 +9,16 @@ on:
       - 'main'
 
 jobs:
-  jammy-ci:
+  noble-ci:
     runs-on: ubuntu-latest
-    name: Ubuntu Jammy CI
+    name: Ubuntu Noble CI
```

Comment:
most of the other packages have added Noble CI alongside Jammy. Should we be disabling that?

Reply:
Since we don't have codecov working in Noble, perhaps we should leave Jammy in for now. Otherwise, I don't think there's a need to keep Jammy.

Reply:
Added jammy back in https://github.com/gazebosim/gz-fuel-tools/pull/427/commits/85e4178fd40262170a361bdf0bb6b47aaa66698a with codecov enabled.